content/post/articles/how-to-build-an-oss-project.md

```diff
@@ -0,0 +1,139 @@
+
+- **Ask for a managed Database** we acknowledge that it might be essential to spin up a database for a few projects. If you think your application needs one [talk to us][7], we probably think it doesn’t, but we are open.
+
+- **Write code for humans**, write code others will understand, and don’t try to prove a point by using nested ternaries or comprehensions, complicated OOP design patterns, or fancy architectures. Good patterns, abstractions, and architectures emerge while writing something functional and straightforward. Code readability and maintenance come first. Keep that in mind.
+
+- **Use automation to enforce Architectural decisions.** You can use [GitHub Actions][8] to add checks to enforce linting, run unit or integration tests, and enforce the use of [conventional commits][9] before the code gets merged.
+
+- **Write tests**. Please do. It will help you better architect your code and maintain it in the future.
+
+#### Deployment
+
+To deploy your application, we recommend these solutions:
+
+- **Use** [**GitHub Pages**][10] if you are working on a simple frontend application. A collection of restaurants, for instance.
+
+- **Docker containers** are the recommended way to package your application. If you have a workflow that generates a Docker image, we can use our artifact management system to deploy your application on our servers.
+
+- **Traditional Apache or APM** deployment, we also provide GitHub Action workflows to deploy your applications on Apache and APM.
+
+We are currently working on improving the automatic deployment of our solutions, and you can read [this architecture draft][11] for our artifact management system. It is already used to deploy **jobsika**, but we will share more about that in the future.
+
+### Conclusion
+
+It is fun to work on new projects, we solve problems in our communities, we learn new things, and work with outstanding engineers. But if we are not careful, this work can quickly lead nowhere, and our projects become a pain to maintain in the future. We hope these few recommendations will help you believe that building an open-source project with a community of passionate developers is possible.
+
+Now feel free to check out our list of [project ideas][12], build a team if necessary and start working on it.
+
+[1]: https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects
+[2]: https://github.com/orgs/osscameroon/projects/4
+[3]: https://firebase.google.com/
+[4]: https://www.mongodb.com/cloud/atlas/lp/try2?utm_source=google&utm_campaign=gs_emea_germany_search_nbnoncompetitor_atlas_desktop&utm_term=firebase&utm_medium=cpc_paid_search&utm_ad=e&utm_ad_campaign_id=12314592846&adgroup=133214220641&gclid=CjwKCAjwrqqSBhBbEiwAlQeqGpMNz9_UotyJFwj_vvReJ2aaDG7ZevkrxlprDrea47UuYpGMVchTRBoC7vkQAvD_BwE
```

Comment:
```suggestion
[4]: https://www.mongodb.com/cloud/atlas/lp/try2
```

I don't think it's fair to our readers to share tracking links.

Reply:
Damn, I completely forgot to change that 🤦‍♂️ .